tests/nginx-tests/nginx-tests/limit_conn_complex.t

```diff
@@ -81,13 +82,9 @@ http_get('/req');
 # limit_req tests
 
 $s = http_get('/req', start => 1);
-SKIP: { 
-skip 'Tengine does not count when the variable arg_r is null ';
```

Comment:
Make test case failed for current Tengine.
When Tengine merged nginx-1.15.x completely, this case will pass.